src/js/renderers/ObjectRenderer.js

```diff
@@ -3,23 +3,23 @@ import {
   findUISchema,
   Generate,
   isObjectControl,
-  RankedTester,
   rankWith,
-  StatePropsOfControlWithDetail,
 } from '@jsonforms/core';
 import { 
   JsonFormsDispatch, 
   withJsonFormsDetailProps,
  } from '@jsonforms/react';
-import { Hidden } from '@mui/material';
```

Comment:
Get rid of @mui since we are using Gutenberg, also `Hidden` component is not supported in mui now
See https://mui.com/material-ui/migration/v5-component-changes/#hidden